src/net_processing.h

```diff
@@ -97,6 +97,6 @@ struct CNodeStateStats {
 bool GetNodeStateStats(NodeId nodeid, CNodeStateStats &stats);
 
 /** Relay transaction to every node */
-void RelayTransaction(const uint256&, const CConnman& connman);
+void RelayTransaction(const uint256& txid, const uint256& wtxid, const CConnman& connman) EXCLUSIVE_LOCKS_REQUIRED(cs_main);
```

Comment:
Looks like this means `cs_main` needs to be locked in `ReattemptInintialBroadcast()` (net_processing) and `BroadcastTransaction()` (node/transaction.cpp)

Reply:
Thanks for catching those, fixed.